cmd/root/bundle_test.go

```diff
@@ -131,8 +142,9 @@ func TestBundleConfigureWithMismatchedProfileEnvVariable(t *testing.T) {
 	t.Setenv("DATABRICKS_CONFIG_PROFILE", "PROFILE-1")
 	cmd := emptyCommand(t)
 
-	err := setupWithHost(t, cmd, "https://x.com")
-	assert.ErrorContains(t, err, "the host in the profile (https://a.com) doesn’t match the host configured in the bundle (https://x.com)")
+	diags := setupWithHost(t, cmd, "https://x.com")
+	require.NotEmpty(t, diags)
```

Comment:
##We should also assert on the length of diagnostics, if we are asserting on diags[0] later. This is because the order of diagnostics is not guarenteed.

Reply:
makes sense, added length checks